recompose-composer/src/main/kotlin/recompose/composer/writer/ModifierBuilder.kt

```diff
@@ -62,12 +62,22 @@ internal class ModifierBuilder(
         val view = node.view
 
         when (view.width) {
-            is LayoutSize.Absolute -> addSize("width", (view.width as LayoutSize.Absolute).size)
+            is LayoutSize.Absolute ->
+                // Don't add the width modifier if it is equal to 0. This is needed to ignore the width parameter in
+                // ConstraintLayout's children when the width is match_constraints (0dp)
+                if (view.width != LayoutSize.Absolute(Size.Dp(0))) {
+                    addSize("width", (view.width as LayoutSize.Absolute).size)
+                }
```

Comment:
Maybe let's additionally check for `view.constraints.hasConstraints()`? If this view has no constraints then I guess it's fine to use `0dp`?

Reply:
Yes, I should add the check for constraints.
One more thing, do you think that the `ModifierBuilder` is the best place for this logic? I myself not sure about this.

Reply:
Yeah, I guess so, since it is also writing the constraints. Eventually it may be something to move out, but maybe not yet?